Src/Service/Service/service/OrganizationService.cs

```diff
@@ -45,6 +45,9 @@ public virtual void RemoveOrganization(int recordId)
                 return;
             }
 
+            IOrganizationUser[] organizationUsers = UnitOfWork.OrganizationUsers.FindBy(ou => ou.Organization == organization);
+            UnitOfWork.OrganizationUsers.RemoveAll(organizationUsers);
+

```

Comment:
Believe we need to do the same thing for Projects here. They're tied to organizations, and shouldn't survive without the organization that created them.